CONTRIBUTING.md

```diff
@@ -152,7 +152,7 @@ and labels mean.
 
 We use GitHub "labels" ([see our
 list](https://github.com/python/mypy/labels)) to roughly order what we
-want to do soon and less soon.  There's two dimensions taken into
+want to do soon and less soon.  There are two dimensions taken into
```

Comment:
"There's" is common in this context in colloquial English but I agree "There are" is better here